### PR TITLE
Add "Stabilizing bitcoin value" page

### LINKED ISSUE #866: Fiat Over Lightning - How it Works
As we discussed [on this call](https://github.com/BitcoinDesign/Meta/issues/323), representing fiat within a bitcoin wallet is becoming important. Price volatility is UX problem for using bitcoin for daily spending, and fiat over LN is one solution for this.

Often this involves things that don't always mesh with the principles of the Guide (Strike, Chivo). Then there are wallets like Bitcoin Beach which are technically custodial but make these trade-offs in a more transparent way. And finally, there are discussions around things like stabilized lightning channels with DLCs, TARO, etc.

This is not specific to merchants, however is important within the conversation about merchant adoption.

With this issue, I am proposing we create content for the [How it Works](https://bitcoin.design/guide/how-it-works/) section that describes at a high-level some of the proposed solutions and what the trade-offs and risks are.

### PR DESCRIPTION
**Pull request by GBKS**
Adds a new page to the How it works section about stabilizing bitcoin value relative to other currencies.

🙅[Check the preview](https://deploy-preview-1067--bitcoin-design-site.netlify.app/guide/how-it-works/stabilizing-bitcoin-value/)🙆

Closes #866

### CODE REVIEW

**Review by rabbitholiness — APPROVED** [review]
Looks good to me. Didn't find anything to nitpick on. I also checked all the links on the page. 👍

**Comment by GBKS** [comment]
@sbddesign may I ask you for your review on this one, since you wrote the original draft?

File: guide/how-it-works/stabilizing-bitcoin-value.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+- Option to have a stable asset
+- The stability is not scoped to a particular app platform, but instead can be implemented by many apps
+- Self-custodial and open-source
+
+{% include fact/close.html %}
+
+{% include fact/cons.html %}
+
+- The oracle goes down, so the smart contract no longer knows what the real-world price is. A default path in the DLC must be exercised and the user reclaims their original quantity of bitcoin but loses the price stability.
```
Comment (bonomat) [inline guide/how-it-works/stabilizing-bitcoin-value.md:237]: As long as Robert and Alice cooperate they don't need the oracle. The oracle is only needed if the two do not cooperate.

File: guide/how-it-works/stabilizing-bitcoin-value.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+- If there is extreme price volatility, exceeding the bounds of what is expressed in the DLC, the user loses stability.
+- In some market conditions, perhaps there are not enough willing counterparties to go around and match the other side of the bet, so users do not have the ability to stabilize new quantities of bitcoin
```
Comment (bonomat) [inline guide/how-it-works/stabilizing-bitcoin-value.md:239]:
```suggestion
- In some extreme market conditions, e.g. during a flash crash, there might not be a counterparty around to match the other side of the bet immediately. In this case, the user might either get a bad deal or risk devaluating their position.
```

File: guide/how-it-works/stabilizing-bitcoin-value.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+However, for a user who hopes to use bitcoin for daily spending or remittances, the volatility is a problem. Suppose Alice lives paycheck to paycheck. If she is paid in bitcoin, even a relatively small decrease in the price could make her unable to pay her Euro-denominated bills each month.
+
+There are times when we prefer to have a stable amount of bitcoin. Other times we prefer a stable amount of another currency (which means that the amount of bitcoin fluctuates in relation).
+
+The most direct and established option is for Alice to simply trade her bitcoin for local currency via an exchange. However, there are various techniques that help to stabilize bitcoin to the dollar value without selling. Some of these techniques are:
+
+- [Tokens](#tokens)
+- [Synthetic currencies](#synthetic-currencies)
+- [Discrete Log Contracts (DLCs)](#discrete-log-contracts-dlcs)
```
Comment (bonomat) [inline guide/how-it-works/stabilizing-bitcoin-value.md:53]: There is one more category which you could write about: Stable Channels see Tony's tweet here: https://x.com/tonklaus/status/1743179494524121559?s=20

**Review by bonomat — COMMENTED** [review]
Nice write up and thanks for mentioning 10101 :)

File: guide/how-it-works/stabilizing-bitcoin-value.md (hunk 1 above)
Comment (GBKS) [reply guide/how-it-works/stabilizing-bitcoin-value.md:237]: That's interesting. I assumed the process was fully automated based on the oracle data, but sounds like the oracle is there for conflict resolution.

So what's the process for Robert and Alice to cooperate on this? One of them requests a "closing" of the contract with a specific outcome, and the other has to agree/sign?

File: guide/how-it-works/stabilizing-bitcoin-value.md (hunk 3 above)
Comment (GBKS) [reply guide/how-it-works/stabilizing-bitcoin-value.md:53]: Thank you. I'll open that up as a separate issue once this PR is merged.

File: guide/how-it-works/stabilizing-bitcoin-value.md (hunk 1 above)
Comment (bonomat) [reply guide/how-it-works/stabilizing-bitcoin-value.md:237]: > So what's the process for Robert and Alice to cooperate on this? One of them requests a "closing" of the contract with a specific outcome, and the other has to agree/sign?

Yes exactly. We call it _collaborative close_. In general it behaves similar to how lightning resolves HTLCs: 
`Look, I know the secret, I could go on-chain and get the money from the HTLC ... or, we could resolve this off-chain` vs. `Look, the oracle would say price is X, I could go on-chain and get the money ... or, we could resolve this off-chain`.
Obviously, if the oracle is not there anymore, a non-cooperative party could keep the money hostage or say you have to go on-chain... and if then there is still on oracle, a refund would come into play and both parties would get their money back.

Comment (GBKS) [reply guide/how-it-works/stabilizing-bitcoin-value.md]: I updated the text (not where you left the comment, but a bit further up the page where oracles are first described). Thanks for the input.

**Comment by GBKS** [comment]
I made a few copy tweaks based on feedback, and also updated from master. Ready for another review. Thanks in advance.

**Review by danielnordh — APPROVED** [review]
Good addition

**Pull request merged**